src/utils/createCodeFrags.ts

```diff
@@ -0,0 +1,17 @@
+import { ICodeFrags, IDtreeSetPoint } from '@service-providers/decision-trees'
+
+export const createCodeFrags = (points: IDtreeSetPoint[]): ICodeFrags[] => {
+  const codeFrags: ICodeFrags[] = []
+
+  points.forEach(
+    (point, index) =>
+      index % 2 === 0 &&
```

Comment:
I think it will be better to use keys from `cond-atoms` instead of even indexes. Or filter by `kind === 'If` which is the same, because `cond-atoms` existed only for `If` points
<img width="308" alt="image" src="https://user-images.githubusercontent.com/98574721/172293727-c5171ac7-10c4-426f-a80b-832ec6a7addd.png">

Reply:
Right, thanks